fastai2/learner.py

```diff
@@ -369,6 +369,18 @@ def accumulate(self, learn):
     @property
     def value(self): return self.val/(1-self.beta**self.count)
 
+# Cell
+class ValueMetric(Metric):
+    "Use to include a pre-calculated metric value (e.g., a metric calculated in a `Callback` and returned in `func`)"
+    def __init__(self, func, metric_name:str=None):
+        store_attr(self, 'func, metric_name')
```

Comment:
I think this can all fit in one line. Also note we have stopped using type annotations (it's pretty obvious `metric_name` should be a string here).